packages/@ama-sdk/create/src/index.it.spec.ts

```diff
@@ -48,16 +48,29 @@ describe('Create new sdk command', () => {
 
   beforeEach(() => {
     cpSync(path.join(__dirname, '..', 'testing', 'mocks', 'MOCK_swagger_updated.yaml'), path.join(sdkFolderPath, 'swagger-spec.yml'));
+    cpSync(path.join(__dirname, '..', 'testing', 'mocks', 'MOCK_DATE_UTILS_swagger.yaml'), path.join(sdkFolderPath, 'swagger-spec-with-date.yml'));
   });
 
-  test('should generate a full SDK when the specification is provided', () => {
+  test('should generate a light SDK when the specification is provided', () => {
     expect(() =>
       packageManagerCreate({
         script: '@ama-sdk',
         args: ['typescript', sdkPackageName, '--package-manager', packageManager, '--spec-path', path.join(sdkFolderPath, 'swagger-spec.yml')]
       }, execAppOptions)
     ).not.toThrow();
     expect(() => packageManagerRun({script: 'build'}, { ...execAppOptions, cwd: sdkPackagePath })).not.toThrow();
+    expect(existsSync(path.join(sdkPackagePath, 'src', 'models', 'base', 'pet', 'pet.reviver.ts'))).toBeFalsy();
+  });
+
+  test('should generate a full SDK when the specification is provided', () => {
```

Comment:
Is that the kind of tests where we want to add generated file comparison on top of the "build" step ?

Reply:
for this test, the idea was to see if we could make sure the projects actually build and some expected files are here. I don't  want to over complexify it as it would be unmaintainable. Making sure that at least one reviver exists is easier to maintain